CHANGELOG.md

```diff
@@ -1,6 +1,35 @@
 Changelog
 =========
 
+[1.3.0] - 2022-07-29
+--------------------
+
+### New Features
+
+- Add customization of port (#67)
+
+* Add customization of port
```

Comment:
There shuold be just one line, and use a `-` itemization for consistency.

That reminds me -- you use squash&merge with multiple-commit PRs, or merge commits a lot. Is that actually intended, or accidental? (Personally I heavily dislike them)

Reply:
> There shuold be just one line, and use a `-` itemization for consistency.

done

> That reminds me -- you use squash&merge with multiple-commit PRs, or merge commits a lot. Is that actually intended, or accidental? (Personally I heavily dislike them)

I prefer squash&merge (which is really a rebase, not an actual merge commit) - I prefer to avoid merge commits.
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-pull-request-commits
squashing makes it easy to correlate - 1 BZ <-> 1 commit <-> 1 PR - and also makes it easy if we need to cherry-pick or if we need to create a patch.

Reply:
Right, it's just awkward for PRs that contain multiple unrelated commits, as they get mushed together, and thus are harder to understand, trace back, revert, bisect on, etc.